promql/testdata/operators.test

```diff
@@ -236,6 +236,10 @@ eval instant at 50m 0 == bool 1
 eval instant at 50m 1 == bool 1
 	1
 
+# bool modifier.
```

Comment:
This is still part of the comparison tests, so you don't need this comment.

Reply:
:+1:  I thought you might say that : updated